### ISSUE TITLE
Different undocumented blog identifier format

### ISSUE THREAD

**Issue by mdashlw** [tumblr/docs#126]
Blog https://www.tumblr.com/rarijackdaily has name of `rarijackdaily`, uuid of `t:r41U3WbkgPsyci1uQtKqfw`, and a different identifier of `z-9h8LWr1Kv4VZVwZ_KZkw`
`await tumblr.apiFetch('/v2/blog/(blogIdentifier)/posts/114205831908/permalink')` supports all three. However, I could not find any information on the format of `z-9h8LWr1Kv4VZVwZ_KZkw`.
I found it on https://rarijackdaily.tumblr.com/post/114205831908/spring-3/embed

**Comment by marcustyphoon** [comment]
Ooh, neat! I'd heard whispers of another blog identifier but my understanding is that it's intentionally not documented or used when possible (probably for reasons like "doesn't stay the same if you change your blog name," if I had to guess).

**Comment by cyle** [comment]
Yes, there is a third unique public blog identifier, which is still around for some legacy purposes. It's not as reliable or used as widely as the proper blog UUID (the one prefixed with `t:`), so we don't document it, because we don't recommend relying on it.

**Comment by mdashlw** [comment]
> Yes, there is a third unique public blog identifier, which is still around for some legacy purposes. It's not as reliable or used as widely as the proper blog UUID (the one prefixed with `t:`), so we don't document it, because we don't recommend relying on it.

I see. Also, what is the format of blog uuids? As far as I can tell, it's not a real uuid

**Comment by cyle** [comment]
> Also, what is the format of blog uuids? As far as I can tell, it's not a real uuid

well, it's a "real" UUID in the sense that it's universally unique _on Tumblr_. it's not a UUID by any of the other standard variants, no. we opted to do something much shorter than the typical UUID v4, for example, because we just didn't need it to be that big.
